debian/mariadb-backup.install

```diff
@@ -1,5 +1,4 @@
-usr/bin/mariabackup
+usr/bin/mariadb-backup
 usr/bin/mbstream
 usr/share/man/man1/mariabackup.1
-usr/share/man/man1/mariadb-backup.1
```

Comment:
Seems you removed the wrong one, you intended to remove mariabackup.1 and keep mariadb-backup.1

Reply:
It's correct. I did NOT change the name of the man pages. I'll save that for the future. mariadb-backup.1 is in .links